__tests__/setup.ts

```diff
@@ -4,9 +4,11 @@ import { createOrGetConnection } from '../src/db';
 import { redisClient, redisPubSub } from '../src/redis';
 
 let con: Connection;
+const excludeEntities = ['ActiveView'];
 
 const cleanDatabase = async (): Promise<void> => {
   for (const entity of con.entityMetadatas) {
+    if (excludeEntities.includes(entity.name)) continue;
```

Comment:
Not a blocker but we should consider checking in the metadata of the entity if it's a view instead of keeping a list of entities to exclude. What do you think?

Reply:
Great suggestion, refactoring that.